src/System.Management/src/System.Management.csproj

```diff
@@ -35,16 +33,26 @@
     <Compile Include="$(CommonPath)\Microsoft\Win32\SafeHandles\SafeLibraryHandle.cs">
       <Link>Common\Microsoft\Win32\SafeHandles\SafeLibraryHandle.cs</Link>
     </Compile>
-    <Compile Include="System\Management\ManagementBaseObject.cs" />
-    <Compile Include="System\Management\ManagementClass.cs" />
+    <Compile Include="System\Management\ManagementBaseObject.cs">
+      <SubType>Component</SubType>
```

Comment:
These shouldn't be necessary.  Presumably VS is just adding them automatically for some reason?

Reply:
Yes. Because ManagementBaseObject inherits from Component. I was going to open an issue against .NET Core build project last week about this but forgot. Will do that tomorrow morning :)

Reply:
Yes, VS keeps adding them back... will delete them again I will see if there is a way to avoid them coming back...

Reply:
> Yes, VS keeps adding them back... will delete them again I will see if there is a way to avoid them coming back...

That is a bug in the .net core  project-system because the designer is not supported in .net core. And that is to tell VS to use the designer when opening that file.